src/scrot.c

```diff
@@ -465,6 +465,7 @@ void scrotGrabMousePointer(const Imlib_Image image, const int xOffset,
 // It assumes that the local variable 'scrot.c:Imlib_Image image' is in context
 static void applyFilterIfRequired(void)
 {
+    warnx("--script is deprecated. Read the man page for details");
```

Comment:
The point of the warning is to see if anyone actually uses this option or not. And to warn them if they do. Doesn't make any sense to keep it around in the manpage if we're planning to remove it.

Just remove the manpage entry entirely and change the warning to:

```suggestion
    warnx("--script is deprecated. See: https://github.com/resurrecting-open-source-projects/scrot/pull/231");
```